app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -268,7 +268,6 @@ class HtmlEditor extends StatefulWidget {
   final bool shrinkWrap;
   final bool disableAutoScroll;
   final EditorState? editorState;
-  final EdgeInsets? editorPadding;
```

Comment:
Found out that this param which was set by default and passed is adding space between newlines. Removed it entirely as it's not to be modified.